batch7crud-roll9/src/main/webapp/WEB-INF/views/detail-view.jsp

```diff
@@ -0,0 +1,30 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: sanjay
+  Date: 1/19/16
+  Time: 2:05 PM
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>SMIS | ${student.getFirstName()} | Details</title>
```

Comment:
Think of the requirement when client ask you to change the language from English to French ? How would you maintain your code ?